addons/survey/models/survey_survey_template.py

```diff
@@ -244,6 +249,13 @@ def action_load_sample_live_presentation(self):
             ]
         }).action_show_sample()
 
+    @api.model
+    def action_load_sample_custom(self):
+        return self.env['survey.survey'].create({
```

Comment:
So this creates an empty survey? Is it useful? Shouldn't we stick with the 3 existing ones that actually fill some data? 

(Or create some questions and title for this one)